lint.whitelist

```diff
@@ -66,6 +66,7 @@ TRAILING WHITESPACE, INDENT TABS, CR AT EOL: *.ani
 TRAILING WHITESPACE, INDENT TABS, CR AT EOL: *.cur
 TRAILING WHITESPACE, INDENT TABS, CR AT EOL: *.ico
 TRAILING WHITESPACE, INDENT TABS, CR AT EOL: *.wasm
+TRAILING WHITESPACE: *.sub.headers
```

Comment:
Just to make sure, is the trailing whitespace necessary for the test? If so, can we exclude just the tests that need it?

Reply:
Yes, the trailing whitespace is part of the test. I could whitelist just those ~3 tests, but the file names could change. 

If you prefer, we could just do:
```
TRAILING WHITESPACE: server-timing/resources/parsing/*.sub.headers
``

Reply:
Yeah, that's probably better